index.html

```diff
@@ -2774,25 +2774,10 @@ <h3>
 
             <p>
 It is RECOMMENDED that dereferencing each <a>URI</a> value of the
-<code>@context</code> property results in a document containing machine-readable
-information about the context. These URIs SHOULD be associated with a
-cryptographic hash of the content of the JSON-LD Context as part of registration
-in the DID Specification Registries [[?DID-SPEC-REGISTRIES]].
-            </p>
-            <p class="advisement">
-If included, the cryptographic hash of the content of the JSON-LD Context MUST
-be computed in a manner equivalent to the following:
-            </p>
-            <pre class="example">
-curl -s https://www.w3.org/ns/did/v1 | openssl sha256
-            </pre>
-            <p>
-The command above will result in the following hash, expressed in hexadecimal
-notation:
-            </p>
-            <pre class="example">
-910cd6648f6f7b72a7896a8da83e63460eb8355a0af4b56b699c9281452ac8bb
-            </pre>
```

Comment:
I'm fairly certain that @OR13 would object to these changes -- he specifically raised an issue and a PR to get this text into the spec. He wants to be crystal clear about how people should generate hashes for JSON-LD Contexts (since we say that's a part of the registration process in DID Spec registries.

Options that might resolve this issue:

1. Don't merge this PR.
2. Merge this PR and put the instructions on how you generate a hash into DID Spec Registries.
3. Move all the language into the Security Considerations section, removing the normative bits.

Reply:
IMO protecting JSON-LD context URLs should not be normative in a spec that has 3 representations... I think all this JSON-LD context integrity protection stuff should be described clearly, one way, in one place, or removed.